### PR TITLE
Fix encoding

### PR DESCRIPTION
**Pull request by tvdijen**
I'm following https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.md as a reference here.

### CODE REVIEW

File: templates/postResponse.twig [hunk 1]
```diff
@@ -6,8 +6,8 @@
     <body>
         <form method="post" action="{{ url }}">
             <input type="hidden" name="wa" value="wsignin1.0">
-            <input type="hidden" name="wresult" value="{{ wresult|escape('html') }}">
-            <input type="hidden" name="wctx" value="{{ wctx|escape('html') }}">
+            <input type="hidden" name="wresult" value="{{ wresult }}">
+            <input type="hidden" name="wctx" value="{{ wctx }}">
```
Comment (thijskh) [inline templates/postResponse.twig:10]: "Remove unnecessary escaping"? HTML escape is just the default in Twig. So this change is sound but a no-op.

Comment (tvdijen) [reply templates/postResponse.twig:10]: According to the cheatsheet you don't have to escape anything in the 'safe' attribute, `value` being one of them.. Am I misunderstanding?

Comment (tvdijen) [reply templates/postResponse.twig:10]: A no-op would be if I changed this to `wresult|escape`, where the method would default to `html`

Comment (thijskh) [reply templates/postResponse.twig:10]: Twig automatically escapes all variables you use. You need to use `|raw` to not to escape a variable.

Comment (tvdijen) [reply templates/postResponse.twig:10]: That I didn't know.. So your concern is the commit msg, not the actual change, right?

Comment (thijskh) [reply templates/postResponse.twig:10]: My 'concern' is indeed that you seemed to be thinking to do one thing while in fact it works differently. In any case the change is sound.

Comment (tvdijen) [reply templates/postResponse.twig:10]: Changed the commit msg

**Pull request merged**